src/importexport/audioexport/internal/audioexportconfiguration.cpp

```diff
@@ -69,6 +69,6 @@ void AudioExportConfiguration::setExportSampleRate(int rate)
 
 const std::vector<int>& AudioExportConfiguration::availableSampleRates() const
 {
-    static const std::vector<int> rates { 32000, 44100, 48000 };
+    static const std::vector<int> rates { 32000, 44100, 48000, 96000, 192000 };
```

Comment:
Shouldn't this go up to 384000 too now?

Reply:
Okay, done. I'm not sure if we should tempt people to use such crazily high sample rates, but on the other hand, if we support it for playback, why not for export, for those people that really want it...